src/Workspaces/Core/Portable/Workspace/Solution/ProjectId.cs

```diff
@@ -28,8 +28,12 @@ private ProjectId(string debugName)
             _debugName = debugName;
         }
 
-        internal ProjectId(Guid guid, string debugName)
+        public ProjectId(Guid guid, string debugName = null)
```

Comment:
[Serialization is a generic word](http://en.wikipedia.org/wiki/Serialization) -- I don't think of it at all meaning the specific serialization technology we're removing support for.